integration/system/ping_test.go

```diff
@@ -50,6 +53,46 @@ func TestPingHead(t *testing.T) {
 	assert.Check(t, hdr(res, "API-Version") != "")
 }
 
+func TestPingSwarmHeader(t *testing.T) {
+	skip.If(t, testEnv.IsRemoteDaemon)
+	skip.If(t, testEnv.DaemonInfo.OSType == "windows")
+
+	defer setupTest(t)()
+	d := daemon.New(t)
+	d.Start(t)
+	defer d.Stop(t)
+	client := d.NewClientT(t)
+	defer client.Close()
+	ctx := context.TODO()
+
+	t.Run("before swarm init", func(t *testing.T) {
+		res, _, err := request.Get("/_ping")
```

Comment:
This should've been:

```suggestion
		res, _, err := request.Get("/_ping", request.Host(d.Sock()))
```

Fixing in https://github.com/moby/moby/pull/43658